site/topics/genetic-algorithms/n-queens-ga-experiment.rst

```diff
@@ -7,6 +7,83 @@ Experimenting with the :math:`n` Queens Genetic Algorithm
 Run Multiple Times and Plot
 ===========================
 
+#. If not done already, download the
+
+    * :download:`Selection Script </../src/selection.py>`
+    * :download:`Crossover Script </../src/crossover.py>`
+    * :download:`Mutation Script </../src/mutation.py>`
+    * :download:`n Queens GA </../src/ga_n_queens.py>`
```

Comment:
Does this course assume python knowledge? Later you mention to use matplotlib, are the students expected to know how to use `pip`, etc?

Reply:
It may at least be worthwhile to just have a note block like > google `pip` to figure out how to install python packages